index.js

```diff
@@ -0,0 +1,15 @@
+let a;
+let b;
+let x;
+
```

Comment:
Be mindful of scope--defining these variables here makes them available to everything on the same level or below them. Simply, they don't need to be declared at all.
```suggestion
```